solr/solrj/src/java/org/apache/solr/client/solrj/impl/CloudSolrClient.java

```diff
@@ -232,32 +232,32 @@ void evictStale() {
    */
   @Deprecated
   public void setRetryExpiryTime(int secs) {
-    this.retryExpiryTime = TimeUnit.NANOSECONDS.convert(secs, TimeUnit.SECONDS);
+    this.retryExpiryTimeNano = TimeUnit.NANOSECONDS.convert(secs, TimeUnit.SECONDS);
   }
 
   protected final StateCache collectionStateCache = new StateCache();
 
   class ExpiringCachedDocCollection {
     final DocCollection cached;
-    final long cachedAt;
+    final long cachedAtNano;
     // This is the time at which the collection is retried and got the same old version
-    volatile long retriedAt = -1;
+    volatile long retriedAtNano = -1;
     // flag that suggests that this is potentially to be rechecked
     volatile boolean maybeStale = false;
 
     ExpiringCachedDocCollection(DocCollection cached) {
       this.cached = cached;
-      this.cachedAt = System.nanoTime();
+      this.cachedAtNano = System.nanoTime();
     }
 
     boolean isExpired(long timeToLiveMs) {
-      return (System.nanoTime() - cachedAt)
+      return (System.nanoTime() - cachedAtNano)
```

Comment:
could we avoid the timeToLiveMs conversion? I see it hardcoded above as 

```
protected volatile long timeToLiveMs = 60 * 1000L;
```
you could apply the same type of refactoring and have it nanos directly

Reply:
i did a pass, but it looked a bit odd, and I'm worried about introducing a bug..  so I backed it out.  But it does point to do we need a pass through Solr codebase and try and get all our time handling variables names and initialization patterns to be the same?